### ISSUE TITLE
Nothing happens when I try to print.

### ISSUE THREAD

**Issue by gkeraunen** [PeterWone/vsc-print#38]
I installed the latest version but nothing happened when I tried to print a selection of text or a whole file. I tried both using the context menu and clicking on the printer icon.

There was no error message or any kind of feedback. Nothing ever displays in a browser window and nothing prints to my printer.

Chrome is my default browser on Windows 7 and I don't need to use an alternate browser so I disabled the default "use alternate browser" setting.

I also added the browser path: C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

Hoping to at least get some indication that the extension is working, I enabled the option: Announce Port Acquisition.

I don't know how this is supposed to get announced but nothing at all happens when I try to print. Not even an error message or anything.

**Comment by PeterWone** [comment]
Post acquisition is announce via the notification system introduced in Windows 8 so I'm not surprised it doesn't work on Windows 7. I will retest the current release on supported platforms and if necessary add a minimum OS version to the README.

**Comment by PeterWone** [comment]
When you say nothing displays in a browser window, which of the following do you mean? 

1. The browser opens but shows a blank page, 
1. The browser fails to open.

**Comment by PeterWone** [comment]
On a fresh build of Win7Pro retail SP1 
1. I installed current builds of Chrome, VS Code and vscode-print extension 0.7.11 
1. I created a folder **sample** and opened it with VS Code
1. I created a file readme.md, typed some markdown into it and saved the file.
1. I clicked the print icon
1. I clicked OK to allow the application to permit connections on the port.
1. I took this screen snap.
![image](https://user-images.githubusercontent.com/5498936/60976121-7ec4c480-a370-11e9-8bb1-a19490843269.png)

Try getting the latest build of the extension. If that doesn't help then there's something wrong with your system.

**Comment by gkeraunen** [comment]
Hi Peter,
I just installed version 0.7.12 but nothing changed.
It also wasn't a browser issue: chrome was already running but the print command in VS Code never caused chrome to come to the foreground and it didn't open a new tab or do anything.

Turned out the problem was that I had not added firewall permission since I didn't know which 'dynamic port' would be used.
So I simply put the same number into the beginning and end of port range: 49203 and added this port to my firewall config.
Now it prints fine.

You might want to add some mention of firewall port configuration to the basic usage instructions.
Thanks,
Greg
